chainerx_build_helper.py

```diff
@@ -37,36 +37,44 @@ def run(self):
             self.build_extension(ext)
 
     def build_extension(self, ext):
+        # Decide the build type: release/debug
+        build_type = os.getenv('CHAINERX_BUILD_TYPE', None)
+        if build_type is not None:
+            # Use environment variable
+            pass
+        elif self.debug:
+            # Being built with `python setup.py build --debug`
+            build_type = 'Debug'
+        elif os.getenv('READTHEDOCS', None) == 'True':
+            # on ReadTheDocs
+            build_type = 'Debug'
+        else:
+            # default
+            build_type = 'Release'
+
```

Comment:
How about checking `assert build_type in ('Debug', 'Release')`?

Reply:
I don't think it's something to be done here.
`cmake` should handle that.

Reply:
Currently, cmake seems not to check it.

Reply:
Hmm, I think that should be addresed separately.
Those who build chainerx_cc directly should also receive an error.